label_studio/io_storages/s3/models.py

```diff
@@ -193,7 +194,13 @@ def save_annotation(self, annotation):
         key = str(self.prefix) + '/' + key if self.prefix else key
 
         # put object into storage
-        s3.Object(self.bucket, key).put(Body=json.dumps(ser_annotation))
+        additional_params = {}
+        if flag_set('fflag_feat_back_lsdv_3958_server_side_encryption_for_target_storage_short', user=annotation.completed_by):
+            additional_params = {'ServerSideEncryption': 'AES256'}
```

Comment:
should it be enabled for all storages now?

Reply:
I think so, since it's enabled by default in S3